src/main/java/duke/ToDoList.java

```diff
@@ -0,0 +1,165 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDos;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * duke.ToDoList class manages all the tasks created by user
+ * @author Shaune Ang
+ */
+public class ToDoList {
+    ArrayList<Task> toDoList;
```

Comment:
Alternatively, a public `getTask` method can be added so that you can modify the tasks in other files.